cloudigrade/account/tests/test_tasks.py

```diff
@@ -441,3 +444,147 @@ def test_run_inspection_cluster_with_too_many_instances(self, mock_boto3):
 
         with self.assertRaises(AwsTooManyECSInstances):
             tasks.run_inspection_cluster([Mock()])
+
+    @patch('account.tasks.persist_aws_inspection_cluster_results')
+    @patch('account.tasks.read_messages_from_queue')
+    def test_persist_inspect_results_no_messages(
+            self,
+            mock_read_messages_from_queue,
+            mock_persist_aws_inspection_cluster_results
+    ):
+        """Assert empty results does not work."""
+        mock_read_messages_from_queue.return_value = []
+        tasks.persist_inspection_cluster_results_task()
+        mock_read_messages_from_queue.assert_called_once_with(
+            settings.HOUNDIGRADE_RESULTS_QUEUE_NAME,
+            tasks.HOUNDIGRADE_MESSAGE_READ_LEN
+        )
+        mock_persist_aws_inspection_cluster_results.assert_not_called()
+
+    def test_persist_aws_inspection_cluster_results_mark_rhel(self):
+        """Assert that rhel_images are tagged rhel."""
+        ami_id = util_helper.generate_dummy_image_id()
+        user1 = util_helper.generate_test_user()
+        account1 = helper.generate_aws_account(user=user1)
+        machine_image1 = \
+            helper.generate_aws_image(account=account1,
+                                      is_encrypted=False,
+                                      is_windows=False,
+                                      ec2_ami_id=ami_id)
+        inspection_results = {'cloud': 'aws',
```

Comment:
Yikes… that indentation. 😯

While this is all technically valid, I wonder if reformatting it would improve readability? This version still fits in the line width rules:

```
        inspection_results = {
            'cloud': 'aws',
            'results': {
                ami_id: {
                    'drive': {
                        'partition': {
                            'rhel_found': True,
                            'evidence': [
                                {
                                    'release_file': '/redhat-release',
                                    'release_file_contents': 'RHEL\n',
                                    'rhel_found': True,
                                }
                            ],
                        }
                    }
                }
            }
        }
```